internal/local/app_test.go

```diff
@@ -673,87 +678,39 @@ exports = function({ query }) {
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
-
-var appData20180301Remote = AppDataV1{AppStructureV1{
-	ConfigVersion:   realm.AppConfigVersion20180301,
-	ID:              "20180301-remote-abcde",
-	Name:            "20180301-remote",
-	Location:        realm.LocationVirginia,
-	DeploymentModel: realm.DeploymentModelGlobal,
-	Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
-	Hosting: map[string]interface{}{
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
-
-var appData20200603Local = AppDataV1{AppStructureV1{
-	ConfigVersion:   realm.AppConfigVersion20200603,
-	Name:            "20200603-local",
-	Location:        realm.LocationVirginia,
-	DeploymentModel: realm.DeploymentModelGlobal,
-	Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
-	Hosting: map[string]interface{}{
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
-
-var appData20200603Remote = AppDataV1{AppStructureV1{
-	ConfigVersion:   realm.AppConfigVersion20200603,
-	ID:              "20200603-remote-abcde",
-	Name:            "20200603-remote",
-	Location:        realm.LocationVirginia,
-	DeploymentModel: realm.DeploymentModelGlobal,
-	Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
-	Hosting: map[string]interface{}{
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
+func createAppDataV1(version realm.AppConfigVersion, name string) AppDataV1 {
```

Comment:
there would have been 6 very similar vars for each AppDataV1 used for testing 
[appData20200603Remote, appData20200603Local, appData20200603Nested
appData20180301Remote, appData20180301Local, appData20180301Nested]

so I replaced them with this `createAppDataV1` that takes in the various versions